### ISSUE TITLE
Modules in a different menuRoot should indicate so

### ISSUE THREAD

**Issue by ctrueden** [scijava/scijava-search#12]
~~In Fiji, search for "fun" and "Fix Funny Filenames" will come up, which is a module intended to appear in the menu of the Script Editor.~~ (Edit: The aforementioned command is not actually an illustration of this issue.)

The SciJava framework supports multiple menu structures via the `UIDetails#getMenuRoot()` value. The `ModuleSearchResult` should include this metadata in its reported menu path when it differs from the default app menu.

**Comment by imagejan** [comment]
I tried to use `info.getMenuRoot()`, but it returns `app` for the *Fix Funny Filenames* module as well as for modules from Fiji's main menu.
Should this be changed in the script editor, so that it reports a different root, @ctrueden?

**Comment by ctrueden** [comment]
Hmm, I was incorrect: "Fix Funny Filenames" is not part of the script editor. I was thinking of "Zap Gremlins" which is a different thing. The `Fix_Funny_Filenames.ijm` script ships with Fiji and _does_ appear in the main application menu as intended.

I think my issue here is still ultimately correct, but actually there are not any commands which currently trigger the issue. I looked for them in `imagej-plugins-commands`:
```
$ git grep menuRoot
src/main/java/net/imagej/plugins/commands/imglib/DuplicateImageContext.java:    menuRoot = ImageDisplay.CONTEXT_MENU_ROOT, headless = true, attrs = { @Attr(name = "no-legacy") })
src/main/java/net/imagej/plugins/commands/restructure/SplitChannelsContext.java:        mnemonic = 's') }, menuRoot = ImageDisplay.CONTEXT_MENU_ROOT, headless = true, attrs = { @Attr(name = "no-legacy") })
```
But as you can see, both of those commands are flagged with the `no-legacy` attribute, which causes them to be excluded from the ImageJ application context when running in legacy mode.

So I'm sorry for the (mostly) noise, @imagejan.

**Comment by ctrueden** [comment]
This was addressed in 55bc0f2d1389c2dcedea3784f55a2260b9e48e2f. It is helpful when e.g. the SciView update site is enabled, since SciView registers its commands under a "SciView" menu root.
